packages/calypso-e2e/src/lib/components/template-part-list-component.ts

```diff
@@ -0,0 +1,51 @@
+import { Locator, Page } from 'playwright';
+
+const parentSelector = '[aria-label="Template parts list - Content"]';
+
+const selectors = {
+	deleteButton: `${ parentSelector } button:has-text("Delete")`,
+	actionsButtonForPart: ( partName: string ) =>
+		`${ parentSelector } .edit-site-list-table-row:has-text("${ partName }") button[aria-label="Actions"]`,
+};
+
+/**
+ * Represents the list of template parts in the full site editor.
```

Comment:
I'm not too familiar with FSE, so I tried to find the component represented by this class and I wasn't able to find it.

What does this (and the other `TemplatePart` class map to?